src/observable/dom/AjaxObservable.ts

```diff
@@ -417,7 +417,7 @@ export class AjaxResponse {
           this.response = JSON.parse(xhr.responseText || 'null');
         }
         break;
-      case 'xml':
```

Comment:
why are we removing `case 'xml'`? This is a breaking change. If this is truly necessary: 1. We'll need demonstratable proof of the need to remove this, and 2. We'll need to wait until the next MAJOR release (RxJS 6) in order to get this into production (because it's a breaking change)